README.md

```diff
@@ -54,32 +54,35 @@ Managed node host:
 * Docker >= 1.12
 * python >= 2.6
 * docker-py >= 1.7.0
+* Java => 8
```

Comment:
this is really needed? Hyperfoil does not run on docker?

Reply:
Using the Ansible integration roles means yes having a dependency on Java.

Reply:
Then creating traffic file requires docker, but running injector client does not. Documentation about host requirements should be updated?

Java can be installed using ansible? or left as requirement?

Reply:
I'll investigate installing java using Ansible in another commit.